example/basic/main.js

```diff
@@ -21,55 +21,80 @@ function login() {
     connectURL: constants.VEZGO_CONNECT_URL,
     clientId: constants.VEZGO_CLIENT_ID,
     baseURL: constants.VEZGO_API_URL || 'https://api.vezgo.com/v1',
-    authEndpoint: "/vezgo/auth",
+    authEndpoint: '/vezgo/auth',
     auth: {
-      headers: { Authorization: `Bearer ${$("#username").val()}` },
+      headers: { Authorization: `Bearer ${$('#username').val()}` },
     },
     // To enable "Demo" provider
-    demo: true
+    demo: true,
   });
-  $("#result").html("");
-  $("#response_heading").html(`Loading... `);
+  $('#result').html('');
+  $('#response_heading').html('Loading... ');
   user = vezgo.login();
 }
 
 // Document ready event
-$(document).ready(function () {
+$(document).ready(() => {
   // Connect button click handler
-  $("#connect").click(function () {
-    $("#connection_error strong").text("");
+  $('#connect').click(() => {
+    $('#connection_error strong').text('');
     login();
+    const multiWallet = window.location.search.includes('multi_wallet=true');
 
     user
       .connect({
+        // provider: 'binance',
+        providers: ['metamask', 'demo', 'coinbase'],
         theme: constants.VEZGO_CLIENT_THEME,
         providersPerLine: constants.VEZGO_CLIENT_PROVIDERS_PER_LINE,
         connectionType: constants.VEZGO_CONNECT_TYPE,
         // providerCategories: ['exchanges', 'wallets'],
+        multiWallet,
       })
-      .onEvent(function (event, data) {
+      .onEvent((event, data) => {
         console.log('event', event, data);
       })
-      .onConnection(function (account) {
-        console.log("connection success", account);
-        $("#response_heading").html(`Account connected successfully with ID: ${account}`);
-        $("#account_id_mod").val(account);
+      .onConnection((result, message, wallet) => {
```

Comment:
same here,  not sure if we still need message and wallet. I think we would have the same single `result` param in both modes, but in single mode result would contain just the account id (no change) and in multi mode result would be the object?

Reply:
Discussed in thread, reverted.